src/main/java/org/embulk/output/sftp/SftpLocalFileOutput.java

```diff
@@ -55,13 +57,14 @@
     BufferedOutputStream remoteOutput; // to keep output stream open during append mode
     long bufLen = 0L; // local temp file size
 
-    SftpLocalFileOutput(PluginTask task, int taskIndex)
+    SftpLocalFileOutput(final PluginTask task, final int taskIndex, final TempFileSpace tempFileSpace)
```

Comment:
Changed this to receive a `TempFileSpace` instance (instead of calling `Exec.getTempFileSpace()` in this class) so that this class can be called from tests.